plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit/edit-template-part.tsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Spinner } from '@wordpress/components';
+import { useEntityBlockEditor, store as coreStore } from '@wordpress/core-data';
+import {
+	InnerBlocks,
+	useInnerBlocksProps,
+	useBlockProps,
+} from '@wordpress/block-editor';
+
+export type FeaturesKeys = 'isBlockifiedAddToCart';
```

Comment:
This could be moved to a conts file as it already exists in `plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit/index.tsx`

Reply:
Ups, yes, actually this wasn't used anywhere. I removed it in 977ab97498b0965ebb69cc0e71862359d8c669c3.